src/components/LayerUI.tsx

```diff
@@ -40,6 +42,24 @@ interface LayerUIProps {
   onLockToggle: () => void;
 }
 
+const renderEncryptedIcon = () => {
+  return (
+    <div className="encrypted-icon tooltip">
+      <span className="tooltip-text">
+        {t("encrypted.prefix")}{" "}
```

Comment:
```suggestion
        {t("encrypted.prefix")}<br>
```

Reply:
huh.. I just did that right before your suggestion.. but forgot to remove the space.

Reply:
Actually, the space was removed by linter.